tests/test_events.py

```diff
@@ -91,95 +91,113 @@ def test_updates_made_false(event, event_document):
-    (3) Set the time to 11:09 am: the meeting should continue, regardless of Legistar.
-    (4) Set the time to 11:10 am: the meeting should continue, because Legistar lists it as "In progress."
-    (5) Set the time to 11:10 am: the meeting should NOT continue, because Legistar does not list it as "In progress."
-
-    This consider cases to determine if a Board Meeting and Crenshaw Project meeting should appear as concurrent, with the Board Meeting first in the queryset:
-    (1) Set the time to 8:55 am (i.e., five minutes before a 9:00 event, when the event should first appear).
-    (2) Set the time to 9:54 am: the events should continue regardless of Legistar.
+    This test ensures that the `calculate_current_meetings` function returns the
+    first committee event, in a succession of events.
+
+    This test considers four cases to determine if the 'System Safety, Security
+    and Operations Committee' should appear as current:
+
+    (1) Set the time to 8:55 am (i.e., five minutes before a 9:00 event, when
+    the event should first appear).
+    (2) Set the time to 9:54 am: the event should continue, regardless of
+    Legistar.
+    (3) Set the time to 10:01 am: the event should continue, because Legistar
+    lists it as "In progress."
+    (4) Set the time to 10:01 am: the event should NOT continue, because
+    Legistar does not list it as "In progress."
+
+    This test considers three cases to determine if the 'Construction Committee'
+    meeting should appear as current:
+
+    (1) Set the time to 10:10 am (i.e., five minutes before a 10:15 event, when
+    the event should first appear). Assume that the previous event ('System
+    Safety') has ended.
+    (2) Set the time to 10:10 am (i.e., five minutes before a 10:15 event, when
+    the event should first appear) - however, assume that the previous event
+    ('System Safety') has NOT ended.
+    (3) Set the time to 11:09 am: the meeting should continue, regardless of
+    Legistar.
+
+    This consider cases to determine if a Board Meeting and Crenshaw Project
+    meeting should appear as concurrent, with the Board Meeting first in the
+    queryset:
+
+    (1) Set the time to 8:55 am (i.e., five minutes before a 9:00 event, when
+    the event should first appear).
+    (2) Set the time to 9:54 am: the events should continue regardless of
+    Legistar.
     '''
-
     planning_meeting_info = {
         'ocd_id': 'ocd-event/4cb9995c-c42f-4eb9-a8b4-f8e135045661',
-        'name': 'Planning and Programming Committee', 
+        'name': 'Planning and Programming Committee',
         'start_time': '2018-01-17 2:00:00',
         'slug': 'planning-and-programming-committee-f8e135045661'
     }
     event.build(**planning_meeting_info)
 
     safety_meeting_info = {
         'ocd_id': 'ocd-event/5e84e91d-279c-4c83-a463-4a0e05784b62',
-        'name': 'System Safety, Security and Operations Committee', 
+        'name': 'System Safety, Security and Operations Committee',
```

Comment:
Does it matter that there is no `slug` here?

Reply:
hmmmm i don't think so